packages/netlify-cms-widget-markdown/src/MarkdownControl/plugins/ForceInsert.js

```diff
@@ -0,0 +1,42 @@
+const ForceInsert = ({ defaultType }) => ({
```

Comment:
Provides logic for creating a new block to focus when clicking above or below the leading and trailing blocks in the editor.